src/controller/taskController.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from "express";
+import * as TaskService from "../service/taskService";
+
+export function getTasks(req: Request, res: Response) {
+  const data = TaskService.getTasks();
+  res.json(data);
+}
+
+export function getTaskById(req: Request, res: Response) {
+  const { id } = req.params;
+  const data = TaskService.getTaskById(id);
+
+  res.json(data);
+}
+
+export function createTask(req: Request, res: Response) {
+  const { body } = req;
+  TaskService.createTask(body);
```

Comment:
Even if Service and Model fails to create task. The response would still be Task Created